src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -11,82 +15,95 @@ type sublabel = {
 
 interface HomeMenuItemProps {
   label: string;
-  amount?: number;
+  amount: number;
   sublabels?: sublabel[];
-  noBadge?: boolean;
-  classes: { root: string; content: string; noAlert: string; sublabel: string; icon: string; grey: string };
+  iconType?: string;
 }
 
-const styles = (theme: Theme) =>
+const styles = makeStyles((theme: Theme) =>
   createStyles({
     root: {
       width: '100%',
       border: '1px solid',
-      borderColor: theme.palette.divider,
+      borderColor: theme.palette.primary.main,
       borderRadius: '6px',
-      padding: '15px 15px',
-      marginBottom: '12px',
+      padding: theme.spacing(2),
+      marginTop: theme.spacing(2),
```

Comment:
This was changed to `marginTop` so the "Financial Summary" section on the Home Screen doesn't have extra space above it